DEV_GUIDE.md

```diff
@@ -308,6 +309,53 @@ You'll see an API response.  If the service_timeout change is effective, the soc
 will continue for 3 minutes.  If `socat` terminates after about 10 seconds, the workaround
 has been applied ineffectively.
 
+## Running system tests locally
+### Prerequisites
+* minikube
+* User must have access to a container registry such as [quay.io](https://quay.io) or [docker.io](https://docker.io).
+     Create a public accessible repository within the registry named `kroxylicious`.
+
+### Environment variables
+* `QUAY_ORG`: in case quay is used and the default image `kroxylicious-developer` is not used, the quay organisation must be indicated here.
```

Comment:
Actually, I think having both variables QUAY_ORG and KROXYLICIOUS_IMAGE_REPO is confusing.  I'd eliminate QUAY_ORG

Reply:
done